airflow/contrib/operators/redis_publish_operator.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.utils.decorators import apply_defaults
+from airflow.contrib.hooks.redis_hook import RedisHook
+from airflow.models import BaseOperator
+
+
+class RedisPublishOperator(BaseOperator):
+    """
+    Publish a message to Redis.
+    :param channel: redis channel to which the message is published (templated)
```

Comment:
Enter is needed here .Otherwise, the documentation will not be generated correctly.

Reply:
@mik-laj - I've added a blank line before the param.  Let me know if you have any other recommendations.  Thanks